go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/blevesearch/blevex
+
+go 1.15
+
+require (
+	github.com/blevesearch/bleve v1.0.14
+	github.com/blevesearch/cld2 v0.0.0-20200327141045-8b5f551d37f5
+	github.com/cznic/b v0.0.0-20181122101859-a26611c4d92d
+	github.com/cznic/mathutil v0.0.0-20181122101859-297441e03548 // indirect
+	github.com/cznic/strutil v0.0.0-20181122101858-275e90344537 // indirect
+	github.com/facebookgo/ensure v0.0.0-20200202191622-63f1cf65ac4c // indirect
+	github.com/facebookgo/stack v0.0.0-20160209184415-751773369052 // indirect
+	github.com/facebookgo/subset v0.0.0-20200203212716-c811ad88dec4 // indirect
+	github.com/golang/protobuf v1.4.3
```

Comment:
Perhaps keep this at v1.3.2, so we're not upgrading the version that blevex uses. Bleve uses v1.3.2.